_posts/2017/2017-12-31-avoid-double-negatives.md

```diff
@@ -48,13 +48,15 @@ end
 
 ## But why?
 
-Ruby’s conditional syntax is ‘truthy’. In Ruby’s case that means `nil` is equivalent to `false`. There is no need to do a negated `#nil?` check on an object as it is redundant as shown in the first two examples.
+Ruby’s conditional syntax is ‘truthy’, meaning `nil` is equivalent to `false`. Therefore a negated `#nil?` check on an object is redundant.
 
-The syntax of `!!` in the third example converts a value to boolean, but it is unnecessary and likely obscures your intention. If you _really_ want to do a `nil` check, you should use `#nil?` instead.
+The syntax of `!!` is two consecutive boolean ‘not’ operators, if takes any value and converts it into `true` or `false`. However, given Ruby's ‘truthy’ conditionals it is unnecessary and adds redundant code.
 
-You should definitely avoid using an `else` block with an `unless`: the `else` block is a double negative of the condition. Instead, switch the `unless` for an logically equivalent `if` and swap the code in the successful conditional block for the code in the `else` block. This might require some refactoring to the condition to make the resulting code easy to understand.
+You should definitely avoid using `else` block with an `unless`. This structure is hard to reason about because the `else` block acts as a double negative of the condition. Instead, replace the `unless` for an logically equivalent `if` and swap the code in the two branches of the conditional.
 
-It is also _much_ harder to reason when using an `unless` conditional with any boolean algebra (`&&` or `||`). It is an easy way to confuse yourself! The conditional `unless one && two` is the same code as `if !one || !two`. In this case the complex `if` condition is an opportunity to refactor the conditional into a well named method so it can be better understood.
+It is also _much_ harder to reason when using an `unless` conditional with any boolean algebra (`&&` or `||`). It is an easy way to confuse yourself! The conditional `unless one && two` is equivalent to `if !one || !two`. Although the logic looks messier I find the positive version easier to reason about.
```

Comment:
I think change it from "I find..." to "people often find..."

FYI, this is the clearest of all of the "But why" paragraphs.

Reply:
Or remove 'I find'.

Reply:
Indeed!

Tell 'em how it is. :P